test/test_encryption.py

```diff
@@ -1753,6 +1764,10 @@ def test_case_8(self):
 
 # https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/tests/README.rst#bypass-spawning-mongocryptd
 class TestBypassSpawningMongocryptdProse(EncryptionIntegrationTest):
+    @unittest.skipIf(
+        os.environ.get("TEST_CSFLE"),
+        "this prose test does not work when CSFLE is " "on a system dynamic library search path.",
```

Comment:
https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/tests/README.rst#8-bypass-spawning-mongocryptd
The note at the top says that you cannot run this test unless you disable CSFLE. The only way to disable CSFLE is to both not specify `csfle_path` and make sure CSFLE is not on any search paths. When `TEST_CSFLE` is specified, CSFLE is definitely on the search path, so we have to skip.